index.html

```diff
@@ -192,7 +192,7 @@ <h1>Get your Ticket for the event now here!</h1>
                                 <i class="icon pe-7s-id icon-large"></i>
                             </div>
 
-                           <div class="col-md-4 col-sm-8">
+                           <div class="col-md-4 col-sm-8 text-middle">
                                 <h1>Check out the schedule with 200+ sessions from speakers around the world from the previous year.</h1>
```

Comment:
Add `class="text-center"` to h1.

Reply:
My bad, actually I have not used bootstrap yet. I will soon update it.
Thanks